### PR TITLE
Feature/milestone 1

### PR DESCRIPTION
**Pull request by lossen**

### CODE REVIEW

File: src/scss/lib/components/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      }
+      &:nth-child(3) {
+        width: 25px;
+        top: calc(60% + 4px);
+      }
+    }
+  }
+}
+
+.navbar-menu_mobile{
```
Comment (tiero) [inline src/scss/lib/components/navbar.scss]: dash and underscore at the same time, it's not consistent.

File: src/scss/lib/components/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    letter-spacing: 0.6px;
+    word-spacing: -2px;
+    margin-left: 41px;
+    &:hover{
+      background-color: transparent;
+      color: $gray-400;
+      transition: 0.3s ease-out;
+    }
+  }
+  .navbar-brand__logo{
```
Comment (tiero) [inline src/scss/lib/components/navbar.scss]: dash/underscore

File: src/scss/lib/components/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 50%;
+    margin: 0 auto;
+    li:first-child {
+      margin: 0;
+    }
+  }
+  .menu-list__item{
```
Comment (tiero) [inline src/scss/lib/components/navbar.scss]: dash/underscore

File: src/scss/lib/components/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        background-color: transparent;
+        outline: none;
+        box-shadow: none;
+        border-color: initial;
+      }
+    }
+  }
+}
+
+.navbar_fixed{
```
Comment (tiero) [inline src/scss/lib/components/navbar.scss]: Use just dash to be consistent with the rest of the codebase (eg. slick)

File: src/scss/lib/components/navbar.scss (hunk 1 above)
Comment (lossen) [reply src/scss/lib/components/navbar.scss]: fixed

File: src/scss/lib/components/navbar.scss (hunk 2 above)
Comment (lossen) [reply src/scss/lib/components/navbar.scss]: fixed

File: src/scss/lib/components/navbar.scss (hunk 3 above)
Comment (lossen) [reply src/scss/lib/components/navbar.scss]: fixed

File: src/scss/lib/components/navbar.scss (hunk 4 above)
Comment (lossen) [reply src/scss/lib/components/navbar.scss]: fixed

**Pull request merged**